src/network/networkd-netdev-vrf.c

```diff
@@ -0,0 +1,53 @@
+/***
+    This file is part of systemd.
+
+    Copyright 2016 Andreas Rammhold <andreas@rammhold.de>
+
+    systemd is free software; you can redistribute it and/or modify it
+    under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.
+
+    systemd is distributed in the hope that it will be useful, but
+    WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with systemd; If not, see <http://www.gnu.org/licenses/>.
+***/
+
+#include <net/if.h>
+//#include <linux/vrf.h>
+
+#include "sd-netlink.h"
+
+#include "missing.h"
+
+#include "networkd-netdev-vrf.h"
```

Comment:
nitpick: so far we only seperated `<>` and `""` headers by an empty line, but did not add extra empty lines any further than that...